doc/gh-pages/overview.md

```diff
@@ -8,7 +8,7 @@ nav_order: 1
 
 ## What is MNE?
 
-MNE-CPP is part of [MNE Project](https://mne.tools/stable/index.html){:target="_blank" rel="noopener"}, a set of tools for neurophysiology analysis encompassing various projects that sprawn from MNE-C, originally written by Matti Hämäläinen.
+The MNE-CPP is part of [MNE Project](https://mne.tools/stable/index.html){:target="_blank" rel="noopener"}, a set of tools for neurophysiology analysis encompassing various projects that spawn from MNE-C, originally written by Matti Hämäläinen.
```

Comment:
MNE-CPP was fine as is, the issue with 'the' was with 'MNE Project'. 'MNE-CPP is part of the MNE Project' or MNE-CPP is part of the MNE community' or maybe 'MNE-CPP is part of MNE'

Reply:
Doh!

Reply:
Done.